controllers/glance_controller.go

```diff
@@ -822,7 +824,38 @@ func (r *GlanceReconciler) ensureRegisteredLimits(
 			ResourceName: lName,
 			DefaultLimit: lValue,
 		}
-		_, err = o.CreateRegisteredLimit(r.Log, m)
+		_, err = o.CreateOrUpdateRegisteredLimit(r.Log, m)
```

Comment:
I tested this call and it works properly in my local environment: I'm able to update existing limits in keystone.